### PR TITLE
Added dump file

### PR DESCRIPTION
**Pull request by ChiragSaxena5437**
Added dump file for the visualizer

### CODE REVIEW

**Review by ChiragSaxena5437 — COMMENTED** [review]
Some errors prevail formatting needed.

File: dump.js [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+      await require("three/examples/js/shaders/SobelOperatorShader.js").catch(() => {});
+    return THREE;
+  }
+
+
+  topojson = require("topojson-client@3")
+
+  import {slider} from "@jashkenas/inputs"
+
+  import {colorPicker} from "@shaunlebron/color-picker"
```
Comment (pulkitvyas08) [inline dump.js:436]: imports in the end?

File: dump.js [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+  .domain(d3.range(200).map(i => (i*6/200) - 3))
+  .interpolate(d3.interpolateHcl)*/
+
+
+  d3 = require('https://d3js.org/d3.v5.min.js', "d3-geo@1", "d3-geo-projection@2")
```
Comment (pulkitvyas08) [inline dump.js:406]: `require` needs to be replaced with `import`

File: dump.js [hunk 3]
```diff
@@ -0,0 +1,436 @@
...
+  
+    for (const P of multilinestring.coordinates) {
+      for (let p0, p1 = vertex(P[0], radius), i = 1; i < P.length; ++i) {
+        geometry.vertices.push(p0 = p1, p1 = vertex(P[i], radius));
+      }
+    }
+    return new THREE.LineSegments(geometry, material);
+  }
+
+  tempAnomalyYearlyRaw = await FileAttachment("gst_ann_1891_last.txt").text()  // Yaha data file hain
```
Comment (pulkitvyas08) [inline dump.js:324]: This needs to be a URL

**Review by pulkitvyas08 — APPROVED** [review]

**Pull request merged**